### ISSUE TITLE
linter: ts type `NoNoninteractiveElementInteractionsConfig` invalid

### ISSUE THREAD

**Issue by Sysix** [oxc-project/oxc#23299]
### What version of Oxlint are you using?

unreleased

### What happened?

```ts
export interface NoNoninteractiveElementInteractionsConfig {
  /**
   * An array of event handler names that should trigger this rule.
   */
  handlers?: string[];
  [k: string]: string[];
}
```

Was generated in `apps/oxlint/src-js/package/config.generated.ts`, but TS is throwing an error.

<img width="942" height="187" alt="Image" src="https://github.com/user-attachments/assets/57ef8930-c97b-4d25-a96e-9440a263c9d0" />

**Comment by camc314** [comment]
😞

**Comment by Sysix** [comment]
@camc314 I am fine to re-add the rule to be skipped for the moment. Maybe there are more rules which have the same pattern. 🤔
